gcloud-java-examples/pom.xml

```diff
@@ -22,6 +22,12 @@
       <artifactId>gcloud-java</artifactId>
       <version>${project.version}</version>
     </dependency>
+      <dependency>
```

Comment:
Before we change the other examples to use the jar file we should fix the problem that it takes forever (well, 30min) to run `mvn package -Dskiptests` currently. That's inconvenient.